extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -141,6 +141,15 @@ describe('amp-analytics', function() {
     return waitForSendRequest(analytics, 100);
   }
 
+  function clearIframePing(config) {
+    for (const t in config.triggers) {
+      if (config.triggers[t].iframePing) {
+        config.triggers[t].iframePing = undefined;
```

Comment:
analytics data can be sent using following methods:
- sendBeacon
- XHR
- Image tag
- IframPing

iframe pings are only allowed to be used in vendor config and have to be approved based on certain criteria. since this test is passing in the vendor config as inline config, the iframePing property needs to be unset before analytics tag will process the request.